server/src/test/java/com/hedera/block/server/BlockStreamServiceIT.java

```diff
@@ -16,8 +16,8 @@
 
 package com.hedera.block.server;
 
-import static com.hedera.block.protos.BlockStreamService.*;
-import static com.hedera.block.protos.BlockStreamService.PublishStreamResponse.*;
+import static com.hedera.block.server.Translator.*;
```

Comment:
We should not be using wildcard imports...

Reply:
Fixed